migrations/1457249988959-add-roles.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// load .env
+require('dotenv').load();
+
+const Roles = require('../server/models/roles');
+const titles = Roles.schema.paths.title.enumValues;
+
+exports.up = function(next) {
+
+  let tasks = titles.map((title) => {
+    let query = {title: title};
+    let update = {
+      title: title
+      //accessLevel: 0 // FIXME: Is there a default accessLevel?
```

Comment:
Sorry, I meant `viewer` role is the one with `accessLevel` 0. I've updated the comment above.
The access level is used to determine whether a user can access a doc or not. 
i.e. an admin user can access all docs, a staff user can access both viewer and staff docs, etc.
Changing the model would break a lot of stuff.
I'm thinking for this the easier option would be to populate also the access levels in the migration script.